ui/apps/platform/src/Containers/Dashboard/PatternFly/Widgets/WidgetOptionsMenu.tsx

```diff
@@ -18,12 +19,18 @@ function WidgetOptionsMenu({ bodyContent }: OptionsMenuProps) {
             bodyContent={bodyContent}
         >
             <Button
+                aria-label="Options"
                 variant="secondary"
                 className="pf-u-mr-sm"
                 icon={<CaretDownIcon />}
                 iconPosition="right"
             >
-                Options
+                <CogIcon
+                    className="pf-u-display-inline"
+                    color={
+                        isChanged ? 'var(--pf-global--link--Color)' : 'var(--pf-global--Color--200)'
```

Comment:
Thinking about pro and con of references to global colors brought a question to mind:
* Does gray suggest disabled?
* What do you think about leaving well enough alone with whatever is the solution to reset, instead of redundant conditional rendering. That leads to question whether cog even needs an explicit color.